docs/online-boutique-asm-manifests/sidecars/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sample to illustrate how to customize the OnlineBoutique namespace, you need to replace the variable ONLINEBOUTIQUE_NAMESPACE in for-namespace/kustomization.yaml before deploying this file.
+# As an example: sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g" for-namespace/kustomization.yaml
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+components:
+- all
+- for-namespace
+#- for-memorystore
```

Comment:
Do we need this? If so can we add a comment about it?

Reply:
I removed it, and created a dedicated optional section about that `for-memorystore` setup.